tests/test_answersfile_templating.py

```diff
@@ -0,0 +1,58 @@
+from pathlib import Path
+
+import pytest
+
+import copier
+
+from .helpers import build_file_tree
+
+
+@pytest.fixture(scope="module")
+def template_path(tmp_path_factory) -> str:
+    root = tmp_path_factory.mktemp("template")
+    build_file_tree(
+        {
+            root
+            / ".copier-answers-{{module_name}}.yml": """\
+                # Changes here will be overwritten by Copier
+                [[ _copier_answers|to_nice_yaml ]]
```

Comment:
Bug with old template markers.

```suggestion
                {{ _copier_answers|to_nice_yaml }}
```